src/styles/blocks/calendar.scss

```diff
@@ -1,47 +1,45 @@
 .calendar {
   display: flex;
+  margin: 0 auto;
+  width: 7 * $square-size + 6 * $square-gap + 2 * $square-paddings;
   flex-wrap: wrap;
-  gap: 1px;
-  width: container-width($square-size, $columns, $sides-paddings);
-  padding: 30px;
+  gap: $square-gap;
+  padding: 30px $square-paddings;
 
-  @each $day, $day-value in $week-days {
-    &--start-day-#{$day} &__day:first-child {
-      margin-left: ($square-size + $square-gap) * $day-value;
+  @each $dayWeek, $numDay in $week {
+    &--start-day--#{$dayWeek}
+    &__day:first-child {
+      margin-left: $numDay * ($square-size + $square-gap);
     }
   }
 
-  @for $amount from 28 through 31 {
-    &--month-length-#{$amount}
-    &__day:nth-child(n + #{$amount + 1}) {
+  @for $monthDays from 28 through 31 {
+    &--month-length--#{$monthDays}
+    &__day:nth-child(n + #{$monthDays + 1}) {
       display: none;
     }
   }
 
   &__day {
     display: flex;
     flex-wrap: wrap;
-    align-content: center;
+    align-items: center;
     justify-content: center;
-
-    @include square($square-size);
-
-    font-size: 30px;
-    text-align: center;
-
+    width: $square-size;
+    height: $square-size;
+    border: 1px solid black;
     background-color: #eee;
-    border: 1px solid #000;
-
-    transition: transform 0.5s;
+    padding: $square-paddings;
+    transition: all 0.5s;
 
     &:hover {
+      background-color: pink;
       cursor: pointer;
```

Comment:
Set cursor prop in the element, not in pseudo-class.

Reply:
and this